scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml

```diff
@@ -587,6 +587,29 @@ Broadcom_DFN-6_2.0x2.0mm:
   num_pins_x: 0
   num_pins_y: 3
 
+Texas_VSON-HR_1.5x2mm_P0.5mm:
+  device_type: 'VSON'
+  size_source: 'http://www.ti.com/lit/ds/symlink/tps62823.pdf'
```

Comment:
include the page in the link:
http://www.ti.com/lit/ds/symlink/tps62823.pdf#page=29

Reply:
Done